src/store/juju/slice.ts

```diff
@@ -243,18 +243,16 @@ const slice = createSlice({
     },
     updateCrossModelQuery: (
```

Comment:
We should remove this error handling from this action/reducer and instead add a `storeCrossModelQueryError` like we do for other fetch/store/error handlers.